### ISSUE TITLE
[Develop] Two icons missing

### ISSUE THREAD

**Issue by rannmann** [aritchie05/EcoCraftingTool#43]
In the 9.5.2 updates, two items were added using random images as placeholders.  These need to be fixed before we can merge into main.

* Elevator Call Post
* Sunflower Item

Both of these are marked "TODO" within the `items.ts` file.

**Comment by rannmann** [comment]
Looks good!  I didn't know the images were already available, thanks :D
